### ISSUE TITLE
Setting the namespace via the package attribute in the source AndroidManifest.xml is no longer supported.

### ISSUE THREAD

**Issue by Ilya-Vys** [LinusU/flutter_web_auth#165]
After bump SDK version to 34, faced that problem. How to fix it?

**Comment by zHElEARN** [comment]
I also encountered this issue with `flutter_web_auth` while using `logto_dart_sdk`.

Here is my solution:

Find the file `%USERPROFILE%\AppData\Local\Pub\Cache\hosted\pub.dev\flutter_web_auth-x.x.x\android\build.gradle`.

Modify it to the following content:

```groovy
group 'com.linusu.flutter_web_auth'
version '1.0-SNAPSHOT'

buildscript {
    ext.kotlin_version = '1.5.31'
    repositories {
        google()
        mavenCentral()
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:7.0.3'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
}

rootProject.allprojects {
    repositories {
        google()
        mavenCentral()
    }
}

apply plugin: 'com.android.library'
apply plugin: 'kotlin-android'

android {
    namespace "com.linusu.flutter_web_auth"
    compileSdkVersion 31

    sourceSets {
        main.java.srcDirs += 'src/main/kotlin'
    }
    defaultConfig {
        minSdkVersion 16
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    lintOptions {
        disable 'InvalidPackage'
    }

    dependencies {
        implementation 'androidx.browser:browser:1.3.0'
    }

    compileOptions {
        sourceCompatibility = JavaVersion.VERSION_1_8
        targetCompatibility = JavaVersion.VERSION_1_8
    }

    kotlinOptions {
        jvmTarget = JavaVersion.VERSION_1_8
    }
}

dependencies {
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
}
```

Then you can rebuild the project successfully.

**Comment by LinusU** [comment]
This should be fixed in [🚢 0.6.0 / 2024-10-07](https://github.com/LinusU/flutter_web_auth/releases/tag/v0.6.0), please test and I can reopen if the issue persist
